dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```
@@ -1068,4 +1061,65 @@ public final Response delete(@Context final HttpServletRequest  request,
         }
     }
 
-}
+    /**
+     * Copies container to the specified host
+     *
+     * @param request
+     * @param response
+     * @param id       id identifier to copy.
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @POST
+    @Path("/{id}/_copy")
+    @JSONP
+    @NoCache
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response copy(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response,
+            @PathParam("id") final String id) throws DotDataException, DotSecurityException {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requestAndResponse(request, response).requiredBackendUser(true)
+                .rejectWhenNoUser(true).init();
+        final User user = initData.getUser();
+        final Host host = WebAPILocator.getHostWebAPI().getCurrentHostNoThrow(request);
+        final PageMode pageMode = PageMode.get(request);
+
+        if (!UtilMethods.isSet(id)) {
+
+            Logger.error(this, "The container id is required");
```

We used to throw an exception rather than return the Response

---

Code has been updated according to the feedback